presto-main/src/test/java/com/facebook/presto/operator/scalar/TestIpPrefixFunctions.java

```diff
@@ -57,4 +61,65 @@ public void testStringIpPrefix()
         assertInvalidCast("IP_PREFIX('64:face:book::10', 24)", "Cannot cast value to IPADDRESS: 64:face:book::10");
         assertInvalidCast("IP_PREFIX('123.456.789.012', 24)", "Cannot cast value to IPADDRESS: 123.456.789.012");
     }
+
+    @Test
+    public void testIpSubnetMin()
+    {
+        assertFunction("IP_SUBNET_MIN(IPPREFIX '1.2.3.4/24')", IPADDRESS, "1.2.3.0");
+        assertFunction("IP_SUBNET_MIN(IPPREFIX '1.2.3.4/32')", IPADDRESS, "1.2.3.4");
+        assertFunction("IP_SUBNET_MIN(IPPREFIX '64:ff9b::17/64')", IPADDRESS, "64:ff9b::");
+        assertFunction("IP_SUBNET_MIN(IPPREFIX '64:ff9b::17/127')", IPADDRESS, "64:ff9b::16");
+        assertFunction("IP_SUBNET_MIN(IPPREFIX '64:ff9b::17/128')", IPADDRESS, "64:ff9b::17");
+        assertFunction("IP_SUBNET_MIN(IPPREFIX '64:ff9b::17/0')", IPADDRESS, "::");
+        assertInvalidCast("IP_SUBNET_MIN(IPPREFIX '64::ff9b::10/48')", "Cannot cast value to IPPREFIX: 64::ff9b::10/48");
+        assertInvalidCast("IP_SUBNET_MIN(IPPREFIX '123.456.789.012/24')", "Cannot cast value to IPPREFIX: 123.456.789.012/24");
```

Comment:
Those 2 test cases do not belong to this test method. Unlike the `IP_PREFIX`, which can take `VARCHAR`  as the input, those 2 test cases not testing the behavior of `IP_SUBNET_MIN`, but the behavior of casting `VARCHAR` to `IPPREFIX`.

Reply:
I added these tests based on Rongrong's earlier comment. But i agree that they do not test the ``IP_SUBNET_MIN``  function. If the argument ``IPPREFIX`` is always valid then we need not test the invalid input cases.